### DISCUSSION TITLE
Mobile/offline

### DISCUSSION THREAD

**Discussion by TechGeekUnited** [DustinBrett/daedalOS#336] [Feature Requests]
Even though DaedalOS is completely in the browser and From what i can tell Clientside there is no way to have an offline copy that can be opened as an html file on the go when you aren't Connect to the World Wide Web; Is there a way this could be implemented? Or at least a way to export some of the apps to be offline (EX: Text Editors, Devtools, WEBSD/Webllm, Boxedwine(Should work), ETC)

**Comment by DustinBrett** [comment]
Thanks very much for this suggestion. Indeed offline support is something that is for sure on the timeline and will happen one day. I have been doing a lot of thinking about it over the years and have big plans around Service Workers and making a PWA. My app is in a unique place to make great use for service workers for various things involving the file system, caching, processing & as a "firewall". Likely later this year or next year I will be implementing this feature from the ground up to do many things, and offline support will be the first use case as it comes along with everything.
